lib/install/management/create.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/tasks"
 )
 
+const (
+	uploadRetryCount = 5
```

Comment:
we have a package for this (pkg/retry) so please use it if retry is the way to go